package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "restaurant-page",
+  "version": "1.0.0",
+  "description": "Restaurant page for webpack practice",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
- [ ] It's important to add scripts building in development as well as production mode, as it gives you more options for debugging. While production mode generates a more optimized version. Check this for [Webpack modes](https://webpack.js.org/configuration/mode/)

![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/scripts_requirement.png)